templates/checkout/_partials/steps/addresses.tpl

```diff
@@ -66,6 +66,12 @@
         }
       </div>
 
+      {if isset($delivery_address_error)}
+        <p class="alert alert-danger js-address-error" name="alert-delivery" id="id-failure-address-{$delivery_address_error.id_address}">{$delivery_address_error.exception}</p>
```

Comment:
Don't put Bootstrap class.
StarterTheme uses these classes https://github.com/PrestaShop/StarterTheme/blob/develop/_dev/css/st/_notifications.styl

And, please, the minimal markup in StarterTheme